molecule/_shared/converge.yml

```diff
@@ -2,10 +2,10 @@
 - name: Converge
   hosts: all
   roles:
-    - role: ansible-consul
+    - role: brianshumate.consul
```

Comment:
Brian is no longer involved since 2020

Reply:
I changed it to `community` (let's see if it works in github-actions)